lib/CXGN/BrAPI/v2/ObservationUnits.pm

```diff
@@ -97,6 +97,8 @@ sub search {
             plot_list=>$observation_unit_db_id,
             limit=>$limit,
             offset=>$offset,
+            # Order by plot_number, account for non-numeric plot numbers
+            order_by=>'NULLIF(regexp_replace(plot_number, \'\D\', \'\', \'g\'), \'\')::int',
```

Comment:
@MFlores2021 @darekbienkowski do you think this is worth keeping in the main BreedBase codebase?  This sets a default sorting order for observation units, which can be helpful in Field Book.

Reply:
Work with @MFlores2021 to test this with the field trial layout Brapp for functionality and performance

Reply:
Add a config flag for this